pkg/kubelet/container/event.go

```diff
@@ -40,7 +40,16 @@ const (
 	NodeNotSchedulable      = "NodeNotSchedulable"
 	StartingKubelet         = "Starting"
 	KubeletSetupFailed      = "KubeletSetupFailed"
+	FailedDetachVolume      = "FailedDetachVolume"
```

Comment:
agreed, since we typically only send events for failed volume mounts (in the past), I think it makes more sense to use volume vs. device...